src/assets/styles/main.css

```diff
@@ -0,0 +1,247 @@
+.container {
+  flex: 0.93;
+  padding: 25px 0 31px 0;
+}
+
+.search-bar {
+  gap: 5px;
+  background-color: white;
+  padding: 5px 0;
+  border-radius: 5px;
+}
+
+.search-bar > label {
+  margin-left: 12px;
+  border-color: var(--grey-fill-color);
+}
+
+.search-bar > input {
+  border: none;
+  font-size: 10px;
+  flex: 0.98;
```

Comment:
same question about using flex: 1 here

Reply:
In both containers I needed the width to be smaller that the parent container in order to follow the Figma design. To avoid fixed width and have natural responsiveness, after trying with different measurements with flex, I noticed that `flex: 0.93` maintained the width I needed for the container while being responsive. For the input, `flex: 0.98` added a width that did not take the full width of the container, which looks good for design, but I also noticed that I could have the same effect with a `padding-left`